scripts/pre_commit_linter.py

```diff
@@ -1869,16 +1986,12 @@ def _check_bad_patterns(
                 python_utils.PRINT('(%s files checked, %s errors found)' % (
                     total_files_checked, total_error_count))
                 python_utils.PRINT(summary_message)
-        process_manager['bad_pattern'] = summary_messages
-        global_stdout.append(stdout)
+        self.process_manager['bad_pattern'] = summary_messages
+        _STDOUT_LIST.append(stdout)
 
     def _check_patterns(self):
         """Run checks relate to bad patterns."""
-        args = (
-            (_FILES['.js'] + _FILES['.ts']), _STDOUT_LIST, self.process_manager)
```

Comment:
There can be some bad pattern lint issues in HTML and python files (maybe got introduced until now) which we may have to fix.

Reply:
Noted, thanks. Let's see what the lint checks say, and I'll update this PR if needed.